CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 ### Added
 - Analyzer - Support STI models.
+- Performance - Request only displayed fields in the records list.
```

Comment:
To be perfect, the count should not use unnecessary `LEFT OUTER JOIN]` :
<img width="1231" alt="screen shot 2016-11-01 at 23 13 16" src="https://cloud.githubusercontent.com/assets/664261/19918513/0fccc010-a089-11e6-8150-cce3ddc027b6.png">